AuthenticatorShared/Core/Platform/Models/Enum/DefaultSaveOption.swift

```diff
@@ -0,0 +1,35 @@
+import Foundation
+
+// MARK: - DefaultSaveOption
+
+/// The default location for saving newly added keys via QR code scan and Manual entry.
+///
+enum DefaultSaveOption: String, Equatable, Menuable {
+    /// Ask where to save a code each time a QR code is scanned.
+    case none
+
+    /// Save the code locally without showing any prompt.
+    case saveLocally
+
+    /// Take the user to the Bitwarden PM app to save the code without prompt.
+    case saveToBitwarden
+
+    /// All of the cases to show in the menu, in order.
+    public static let allCases: [Self] = [
+        .saveToBitwarden,
+        .saveLocally,
+        .none,
+    ]
+
+    /// The name of the value to display in the menu.
+    var localizedName: String {
+        switch self {
+        case .none:
+            Localizations.none
```

Comment:
🤔 I'm a bit worried that the user won't understand that "None" means "Ask me every time where to save the item".

Reply:
I think the idea is that you're setting a default option here. So "none" = no default = ask me. But I can double check with Emily on that. 👍

Reply:
I followed up with Emily and she thinks "None" is the best choice. She said changing the label for that option to something like “Ask me” might be less clear than “none” for a user who hasn’t been “asked” yet via our first-time prompt.